architecture/additional_concepts/storage.adoc

```diff
@@ -449,6 +449,15 @@ Fibre Channel and HostPath persistent volumes do not support mount options.
 ====
 endif::openshift-enterprise,openshift-origin[]
 
+=== Recursive chown
+
+Each time a persistent volume is mounted to a Pod, or when a Pod restart occurs, the volume's ownership and permissions are recursively changed to match those of the Pod. Group read/write access is added to the volume when `chown` is performed on all files and directories in the PV. This enables the process running inside the Pod to access the PV file system.
```

Comment:
I would also mention that SELinux labels are changed recursively too.

User could avoid owner changes by not specifying `fsGroup` in their Pods / SCCs, however, they can't avoid SELinux relabeling.

Reply:
@jsafrane Thank you for your feedback. I updated the PR with the suggested information. PTAL. Thanks!